code/game/objects/items/contraband.dm

```diff
@@ -80,7 +80,9 @@
 		if(!user.check_has_mouth()) // We dont want dionae or adherents doing lines of cocaine. Probably.
 			to_chat(SPAN_WARNING("Without a nose, you seem unable to snort from \the [src]."))
 			return TRUE
-
+		if (user.wear_mask)
+			to_chat(user, SPAN_WARNING("Your [user.wear_mask.name] is in the way."))
```

Comment:
This is actually one I've been meaning to poke at and this would need verifying, but I think this would be a more comprehensive check. Checks for helmets that block the face, as well as checks if the mask actually blocks, ie a surgical mask pulled down will not
```suggestion
		if ((user.wear_mask && (user.wear_mask.body_parts_covered & FACE)) || (user.head && (user.head.body_parts_covered & FACE)))
			to_chat(user, SPAN_WARNING("You need to expose your face first."))
```